flexmeasures/api/v3_0/tests/test_api_v3_0_users.py

```diff
@@ -285,12 +285,12 @@ def test_logout(client, setup_api_test_data, requesting_user):
 
 
 @pytest.mark.parametrize(
-    "requesting_user, expected_status_code",
+    "requesting_user, expected_status_code, user_to_update, expected_role",
     [
-        ("test_prosumer_user_2@seita.nl", 403),  # account-admin user
-        ("test_admin_reader_user@seita.nl", 403),  # admin reader user
-        ("test_consultant@seita.nl", 403),  # consultant user
-        ("test_admin_user@seita.nl", 200),  # admin user
+        ("test_prosumer_user_2@seita.nl", 403, 8, 3),  # account-admin user
+        ("test_admin_reader_user@seita.nl", 403, 8, 3),  # admin reader user
+        ("test_consultant@seita.nl", 403, 8, 3),  # consultant user
+        ("test_admin_user@seita.nl", 200, 8, 3),  # admin user
```

Comment:
Thanks for adding requesting users and comments about them.
Please describe (in docstring) who user 8 is and what role 3 is.

The role certainly plays a role in this test, and why it would not be allowed to change it.
Maybe also test another role which is being set. I'm not sure how many lines in `can_modify_role()` are being tested now.

Maybe also test to send more than one role.